scripts/linux/wayland_enable.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+################################################################################
+# This script will enable wayland windowing system
+################################################################################
+
+if [[ "${UID}" != 0 ]]; then
+    (>&2 echo "Error:  $0 must be run as root")
+    exit 1
+fi
+
+enable_wayland() {
+    case ${linuxType} in
+        ubuntu|\"rocky\"|\"centos\")
+            `sed -i 's/WaylandEnable=.*/#WaylandEnable=false/' /etc/gdm3/custom.conf`
+            ;;
+        debian)
+            `sed -i 's/WaylandEnable=.*/#WaylandEnable=false/' /etc/gdm3/daemon.conf`
```

Comment:
Shouldn't this here be true ?

Reply:
Not sure, the comment given in the conf file says that the line **_WaylandEnable=false_** need be commented and uncommented for the wayland enabling and disabling.